api/swagger.yaml

```diff
@@ -445,6 +445,9 @@ definitions:
       OomKillDisable:
         description: "Disable OOM Killer for the container."
         type: "boolean"
+      Init:
+        description: "Run an init inside the container that forwards signals and reaps processes"
+        type: "boolean"
```

Comment:
Thanks for spotting this!

Looking at the code:

https://github.com/moby/moby/blob/bb6fc72cd2bd0f194940c7710d8ed8cf3fb99d21/api/types/container/host_config.go#L383-L384

We may want to add that

- the default (as configured on the daemon) is used if this field is omitted / null
- that the field is nullable ([`x-nullable: true`](https://github.com/moby/moby/blob/28362f11d73b5b72ff9d7eba27cf9f817ec86ef8/api/swagger.yaml#L934))
- perhaps mention something like "this field is omitted if empty" (or "this field is omitted if not set")